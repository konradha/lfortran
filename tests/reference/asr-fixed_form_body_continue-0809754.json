{
    "basename": "asr-fixed_form_body_continue-0809754",
    "cmd": "lfortran --fixed-form --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/fixed_form_body_continue.f",
    "infile_hash": "30660763a27adf72bf2ae2974102a58ffaa28ed14381a10a7b985596",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-fixed_form_body_continue-0809754.stdout",
    "stdout_hash": "12120563c5a26b58ed7aeb5a1e9a69faa59c1fd2e77b82d2df15fd04",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}