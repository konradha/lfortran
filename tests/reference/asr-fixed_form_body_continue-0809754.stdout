(TranslationUnit (SymbolTable 1 {main: (Program (SymbolTable 2 {a: (Variable 2 a Local () () Default (Integer 4 []) Source Public Required .false.)}) main [] [(If (IntegerCompare (IntegerConstant 1 (Integer 4 [])) GtE (IntegerConstant 0 (Integer 4 [])) (Logical 4 []) (LogicalConstant .true. (Logical 4 []))) [(GoTo 20)] []) (= (Var 2 a) (IntegerConstant 1 (Integer 4 [])) ()) (GoTo 30) (GoToTarget 20) (= (Var 2 a) (IntegerConstant 0 (Integer 4 [])) ()) (GoToTarget 30)])}) [])
